### DISCUSSION TITLE
Documentation - website link

### DISCUSSION THREAD

**Discussion by LucieContamin** [hubverse-org/.github#22] [General]
I noticed that multiple repositories or package have documentation linking to the hubverse website using the "old" link ([hubdocs.readthedocs.io](http://hubdocs.readthedocs.io/)). The link automatically redirect, so it's not an emergency, but should we update the link to the "new" link [hubverse.io](http://hubverse.io/) ?

**Comment by zkamvar** [comment]
I created a flood of pull requests to do so. Roughly a third of them have been merged as of right now. 

https://github.com/search?q=org%3Ahubverse-org+author%3Azkamvar+URL&type=pullrequests

**Reply by zkamvar** [reply]
These were most urgent for R packages that were going to be sent to CRAN because the old URL returns a [302 redirect](https://www.keanu.codes/?code=302)
